modules/core/src/test/scala/com/snowplowanalytics/iglu/schemaddl/parquet/CasterSpec.scala

```diff
@@ -277,7 +279,8 @@ class CasterSpec extends org.specs2.Specification { def is = s2"""
       json"""{"XYZ": 42, "xyz": "invalid"}""" -> StructValue(List(NamedValue("xyz", IntValue(42)))),
       json"""{"xyz": null, "XYZ": "invalid"}""" -> StructValue(List(NamedValue("xyz", NullValue))),
       json"""{"XYZ": null, "xyz": "invalid"}""" -> StructValue(List(NamedValue("xyz", NullValue))),
-      json"""{"XYZ": "invalid"}""" -> StructValue(List(NamedValue("xyz", NullValue))),
+      json"""{"XYZ": "invalid"}""" -> StructValue(List(NamedValue("xyz", NullValue))), //Should be error? 
```

Comment:
> Should be error?

If line 280 and 281 are not errors, then I think it's ok for this one to also be not an error.

But I'm not strongly opinionated on this one, especially as it's a really weird edge case scenario, which we may never see in production.  The important thing is: the loader doesn't break.  And we've achieved that.